systemtests/scripts/run_test_kubernetes.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+CURDIR=`readlink -f \`dirname $0\``
+source ${CURDIR}/test_func.sh
+
+ENMASSE_DIR=$1
+SYSTEMTESTS=$2
+TESTCASE=$3
+failure=0
+
+export OPENSHIFT_URL=${OPENSHIFT_URL:-https://localhost:8443}
+export OPENSHIFT_USER=${OPENSHIFT_USER:-test}
+export OPENSHIFT_PASSWD=${OPENSHIFT_PASSWD:-test}
+export OPENSHIFT_PROJECT=${OPENSHIFT_PROJECT:-enmasseci}
+export OPENSHIFT_TEST_LOGDIR=${OPENSHIFT_TEST_LOGDIR:-/tmp/testlogs}
+export OPENSHIFT_USE_TLS=${OPENSHIFT_USE_TLS:-true}
+export ARTIFACTS_DIR=${ARTIFACTS_DIR:-artifacts}
+export DEFAULT_AUTHSERVICE=standard
+export USE_MINIKUBE=true
+
+SANITIZED_PROJECT=$OPENSHIFT_PROJECT
```

Comment:
It's not possible to do that in one command?

Reply:
Probably. parts of this file is common with run_test_components.sh so I will try to refactor it eventually.